content/questions/actions/question-092.md

```diff
@@ -0,0 +1,13 @@
+---
+archetype: "questions"
+title: "Question 092"
+question: "In a GitHub Actions workflow, how do you specify a specific version of Node.js to use in a job?"
+draft: false
+---
+
+
+> https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs#specifying-the-nodejs-version
+1. [x] uses: actions/setup-node@v1 with node-version
+1. [ ] uses: actions/node-setup@v1 with node-version
+1. [ ] uses: setup-node@v1 with version
+1. [ ] uses: setup-node@v1 with node
```

Comment:
Use yaml syntax for the answers, see [example](https://github.com/FidelusAleksander/ghcertified/blob/ea5b9f511f92bfad159f0317b235aa92bc60e81e/content/questions/actions/question-015.md?plain=1#L11)

Also change v1 to v4 or remove version entirely

Reply:
Thanks .. first time to learn about injecting yaml syntax in the md